<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShareBookcase</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>501</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_ST">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_T">
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#0000ff;&quot;&gt;ShareBookcase&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;		       &lt;/span&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-size:16pt; font-style:italic; color:#ff0000;&quot;&gt; Share your books!&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; font-style:italic; color:#ff0000;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-style:italic; color:#000000;&quot;&gt;Design by Hub Zhang(20171684310178)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_M"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_S">
       <item>
        <widget class="QPushButton" name="Sign_In">
         <property name="text">
          <string>Sign In</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Sign_Up">
         <property name="text">
          <string>Sign Up</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>501</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_SS">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_S">
       <item>
        <widget class="QLineEdit" name="kwdEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="Search">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="locale">
        <locale language="Chinese" country="China"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBegin">
    <property name="title">
     <string>Begin</string>
    </property>
    <addaction name="actionSign_In"/>
    <addaction name="actionSign_Up"/>
    <addaction name="actionSign_Out"/>
    <addaction name="separator"/>
    <addaction name="actionUser"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionAdd_Book"/>
    <addaction name="actionTransaction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUser_Guideline"/>
    <addaction name="separator"/>
    <addaction name="actionReport_Bug"/>
    <addaction name="actionConnect_with_us"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionShare"/>
   </widget>
   <addaction name="menuBegin"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSign_Up">
   <property name="text">
    <string>Sign Up</string>
   </property>
  </action>
  <action name="actionSign_In">
   <property name="text">
    <string>Sign In</string>
   </property>
  </action>
  <action name="actionUser">
   <property name="text">
    <string>Guests</string>
   </property>
  </action>
  <action name="actionUser_Guideline">
   <property name="text">
    <string>User Guideline</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionReport_Bug">
   <property name="text">
    <string>Report Bug</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionConnect_with_us">
   <property name="text">
    <string>Connect with us</string>
   </property>
  </action>
  <action name="actionSign_Out">
   <property name="text">
    <string>Sign Out</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_Book">
   <property name="text">
    <string>Add Book</string>
   </property>
  </action>
  <action name="actionTransaction">
   <property name="text">
    <string>Transaction</string>
   </property>
  </action>
  <action name="actionShare">
   <property name="text">
    <string>Share</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Sign_Up</sender>
   <signal>clicked()</signal>
   <receiver>actionSign_Up</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Search</sender>
   <signal>clicked()</signal>
   <receiver>actionSearch</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Sign_In</sender>
   <signal>clicked()</signal>
   <receiver>actionSign_In</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
